samples/apps/copilot-chat-app/webapi/CopilotChatApi.csproj

```diff
@@ -14,6 +14,14 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.SemanticKernel" Version="0.13.442.1-preview" />
```

Comment:
This will force us to have the core team release new nugets whenever we make changes to their code.  If you are comfortable with that, proceed.

I'm just wondering whether we want to do this now or after Build, when there should be less churn...

Reply:
Correct - we've stabilized on our SK dependencies at the moment, so I was hoping we could try this out. We can always switch back to a project reference if we need to. This will also have the side effect of us being able to validate changes to the kernel before we take a dependency on them.